### PR TITLE
Add quickstart doc links

### PR DESCRIPTION
**Pull request by gvelez17**
## Description

A developer should be able to start and upgrade a node in < 5 min

Also DX usability testers need quick documentation to complete their tasks in < 90 min

see https://linear.app/3boxlabs/issue/MET-1622/quickstart-and-upgrade-readmes

## How Has This Been Tested?

Describe the tests that you ran to verify your changes. Provide instructions for reproduction.

- [ ] Run manually

## PR checklist

Before submitting this PR, please make sure:

- [ x] I have tagged the relevant reviewers and interested parties
- [ x] I have updated the READMEs of affected packages
- [ x] I have made corresponding changes to the documentation

## References:

Please list relevant documentation (e.g. tech specs, articles, related work etc.) relevant to this change, and note if the documentation has been updated.

### CODE REVIEW

File: docs-dev/UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Note that although this references the CLI package, the dependencies will cause the core and other server packages to update.
+
+3) restart the ceramic daemon
+
+Depending on how you have configured your server, this may involve a command such as
+
+`systemctl stop ceramic && systemctl start ceramic`
+
+Note this presumes that you have used systemctl startup scripts
+
```
Comment (gvelez17) [inline docs-dev/UPGRADING.md:24]: hey @kammerdiener can we get a copy of one of your systemctl startup scripts or where did those come from? 

 @3benbox what would be the one line command to upgrade the digitalocean kubernetes based install?  will it automatically upgrade like every time it restarts or?

File: docs-dev/QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Configuration
+
+The default configuration file is located at 
+
+`$HOME/.ceramic/daemon.config.json`
+
+For more information about the configuration options see [Server Configurations](https://composedb.js.org/docs/0.5.x/guides/composedb-server/server-configurations)
+
+For an example how to run in the cloud using Digital Ocean and Kubernetes see [Running in the Cloud](https://composedb.js.org/docs/0.4.x/guides/composedb-server/running-in-the-cloud)
```
Comment (JustinaPetr) [inline docs-dev/QUICKSTART.md:28]: I seems like we are pointing to the 0.5x version of the ComposeDB here. Can we use the latest on here and point to the 0.5.x just like int the link right above this one?

**Review by JustinaPetr — CHANGES_REQUESTED** [review]

File: docs-dev/UPGRADING.md (hunk 1 above)
Comment (gvelez17) [reply docs-dev/UPGRADING.md:24]: cool would it work with `npx @ceramicnetwork/cli daemon` without the version pin?  That way they could update without editing the script?

is npx installed with node or its a separate install?  ha sorry for lazy questions but just wanted to give docs for how to set up like you have it on hirenodes!

Maybe we should mention the ulimit too now that i think of it - that might be a separate doc for server-config-example ...

File: docs-dev/QUICKSTART.md (hunk 2 above)
Comment (gvelez17) [reply docs-dev/QUICKSTART.md]: fixed!

File: docs-dev/UPGRADING.md (hunk 1 above)
Comment (3benbox) [reply docs-dev/UPGRADING.md:24]: When running with systemd, we should not use the root account. Any security issues with the ports we expose could escalate to a host compromise.

My ugly unit file is templated with Ansible in the simpledeploy repo
https://github.com/ceramicstudio/simpledeploy/blob/main/ansible/roles/ceramic/templates/ceramic.service.j2

Comment (gvelez17) [reply docs-dev/UPGRADING.md:26]: ok cool - linked them to SimpleDeploy and pointed out that our upgrade example is just a simplified example.

**Comment by gvelez17** [comment]
hey @JustinaPetr i think we have addressed the changes now!

**Review by JustinaPetr — APPROVED** [review]

**Pull request merged**